doc/sphinx/source/algorithms.rst

```diff
@@ -88,7 +88,7 @@ These all assume that an object of :class:`.GraphDataScience` is available as `g
 
 
 .. py:function:: gds.alpha.graph.sample.rwr(graph_name: str, from_G: Graph, **config: Any)
-    -> Tuple[Graph, Series[Any]]
+    -> GraphResult
```

Comment:
If we do this, it needs to be exported as well

Reply:
what do you mean with export here?

Reply:
It should be put in the root `__init__.py` I think. Although maybe we don't need that. It's just that `graph` is an internal namespace

Reply:
we agreed that there are more classes to expose there. Will do a follow up PR